app/src/main/res/layout/notifications_list_item.xml

```diff
@@ -9,12 +10,30 @@
         android:orientation="vertical"
         android:padding="8dp">
 
-        <TextView
-            android:id="@+id/title"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:textAppearance="@style/NavView.TextView.Medium"
-            tools:text="Dzisiaj 1 to szczęśliwy numerek" />
+            android:orientation="horizontal"
+            android:gravity="center_vertical">
+
+            <com.mikepenz.iconics.view.IconicsImageView
+                android:id="@+id/notificationIcon"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:scaleType="centerInside"
+                app:iiv_color="?android:textColorSecondary"
+                app:iiv_icon="cmd-chevron-down"
+                app:iiv_size="24dp"
+                tools:src="@android:drawable/ic_menu_more" />
+
+            <TextView
+                android:id="@+id/title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:paddingStart="8dp"
```

Comment:
You have to also add `paddingLeft` or replace with `marginStart` AND `marginLeft` to ensure compatibility with Android 4.1.